src/execution/operator/persistent/buffered_csv_reader.cpp

```diff
@@ -172,6 +174,34 @@ static string GenerateColumnName(const idx_t total_cols, const idx_t col_number,
 	return string(prefix + leading_zeros + value);
 }
 
+static string NormalizeColumnName(const string &col_name) {
+	// make lowercase
+	const string col_name_lower = StringUtil::Lower(col_name);
+	// remove accents
+	auto stripped = utf8proc_remove_accents((const utf8proc_uint8_t *)col_name_lower.c_str(), col_name_lower.size());
```

Comment:
Perhaps remove accents before doing the lower-case conversion? The `StringUtil::Lower` is not unicode aware, so otherwise you might have `Á` -> `A` and end up with an uppercase character in the final name anyway.

The reason this seems to work in the example below is that DuckDB can do lowercase -> uppercase matching now as long as there are no ambiguities, e.g.:

```sql
-- table with uppercase name
create table integers("HELLO" integer);
insert into integers values (1);
-- this works!
select hello from integers;
-- print the actual name
select alias(hello) from integers;
-- HELLO
```

With ambiguities this is not allowed though:
```sql
-- hello with different cases
create table integers2("HELLO" integer, "hEllO" integer);
-- error! ambiguity between HELLO and hEllO
select hello from integers2;
-- OK! match case exactly
select "HELLO" from integers2;
```

Reply:
Thanks, I would say then it should even be done after throwing out all UTF-8 characters. Did this now!